lib/network/modules/LayoutEngine.js

```diff
@@ -505,6 +505,9 @@ class LayoutEngine {
           }
         }
       }
+      if (allOptions.physics && !allOptions.physics.wind) {
+        allOptions.physics.wind = { x: 0, y: 0 };
+      }
```

Comment:
What is the purpose of this? Reading through the code I just don't see anything this would change? Plus why is layout engine involved in physics? Does it interfere with something? If so there definitely should be a comment explaining why this is.

Reply:
Removed